doc/source/security/verifying-encryption.rst

```diff
@@ -0,0 +1,101 @@
+..  _verifying-encryption:
+
+===================================================================
+Verifying the Encryption for Tables, Tablespaces, and Schemas
+===================================================================
+
+If a general tablespace contains tables, check the table information to see if the table is encrypted. 
+
+When the general tablespace contains no tables, you may verify if the tablespace is encrypted or not. 
+
+For single tablespaces, verify the ENCRYPTION option using `INFORMATION_SCHEMA.TABLES` and the `CREATE OPTIONS` settings. 
+
+.. code-block:: MySQL
+
+    mysql> SELECT TABLE_SCHEMA, TABLE_NAME, CREATE_OPTIONS FROM INFORMATION_SCHEMA.TABLES WHERE CREATE_OPTIONS LIKE '%ENCRYPTION%';
+
+    +----------------------+----------------------+------------------------------+
+    | TABLE_SCHEMA         | TABLE_NAME           | CREATE_OPTIONS               |
+    +----------------------+----------------------+------------------------------+
+    | sample               | t1                   | ENCRYPTION="Y"               |
+    +----------------------+----------------------+------------------------------+
+
+
+A ``flag`` field in the ``INFORMATION_SCHEMA.INNODB_TABLESPACES`` has bit
+number 13 set if tablespace is encrypted. This bit can be ckecked with
+``flag & 8192`` expression in the following way:
+
+.. code-block:: mysql
+
+    SELECT space, name, flag, (flag & 8192) != 0 AS encrypted FROM INFORMATION_SCHEMA.INNODB_TABLESPACES WHERE name in ('foo', 'test/t2', 'bar', 'noencrypt');
+
+.. admonition:: Output
+
+   .. code-block:: guess
+
+      +-------+-----------+-------+-----------+
+      | space | name      | flag  | encrypted |
+      +-------+-----------+-------+-----------+
+      |    29 | foo       | 10240 |      8192 |
+      |    30 | test/t2   |  8225 |      8192 |
+      |    31 | bar       | 10240 |      8192 |
+      |    32 | noencrypt |  2048 |         0 |
+      +-------+-----------+-------+-----------+
+      4 rows in set (0.01 sec)
+
+Encrypted table metadata is contained in the INFORMATION_SCHEMA.INNODB_TABLESPACES_ENCRYPTION table. You must have the ``Process`` privilege to view the table information.
```

Comment:
This is part of encryption threads and should be marked as experimental. Also could you mention that this table is experimental and likely to change in the future releases ?

Reply:
Added a note that emphasized the table is experimental and may change in future releases.